cou/apps/base.py

```diff
@@ -411,19 +411,19 @@ def post_upgrade_steps(self, target: OpenStackRelease) -> list[PostUpgradeStep]:
     def generate_upgrade_plan(
         self,
         target: OpenStackRelease,
+        force: bool,
```

Comment:
Can we use default value to `False`?

Reply:
We can, but I prefer not to so it forces passing the CLI parameter for it.

Reply:
Why? What is the benefit of such approach?

Reply:
Discussed and we will change it latter, if we do not forget.